examples/plotting/simple_differential_rotation.py

```diff
@@ -23,21 +23,22 @@
 
 import sunpy.data.sample
 import sunpy.map
-from sunpy.physics.differential_rotation import diff_rot, solar_rotate_coordinate
+from sunpy.physics.differential_rotation import solar_rotate_coordinate
```

Comment:
This is a big aside, is there a case for this being in sun.models as well?

Reply:
Per #7161, I envision that function eventually moving to `sunpy.coordinates.utils`